make distcheck fails

make distcheck fails because of the missing file js/gresource.xml
```
make[4]: Entering directory '/home/pawel/liferea/liferea-1.13.6/_build/sub/src'
make[4]: *** No rule to make target '../../../js/gresource.xml', needed by 'js.c'.  Stop.
```
The file has not been added because the entry in the Makefile.am file is missing

--- a/Makefile.am
+++ b/Makefile.am
@@ -30,6 +30,7 @@ css_DATA = \
 
 jsdir = $(pkgdatadir)/js
 js_DATA = \
+	js/gresource.xml \
 	js/htmlview.js \
 	js/Readability.js
 

